authentication/views.py

```diff
@@ -56,6 +56,20 @@ def post(self, request):
                 'email': email
             })
 
+class RelateUserToCompany(APIView):
+    """Binding user to company and inserting linked company's id into token."""
+    permission_classes = (IsAuthenticated,)
+    
+    def post(self, request):
+        user_id = request.user.user_id
+        company_id = request.data['company_id']
+        relation = CompanyAndUserRelation.get_relation(user_id, company_id)
+        if not relation: 
+            return Response({'error': 'You have no access to this company.'}, status=status.HTTP_401_UNAUTHORIZED)
```

Comment:
Looks like the HTTP `403 Forbidden` status code might be more appropriate here, as `401 Unauthorized` is typically used when authentication is required but has failed or has not yet been provided

Reply:
Agree, fixed